lib/Settings.cs

```diff
@@ -1,80 +1,262 @@
-﻿namespace Microsoft.ContentAuthenticity;
+﻿
+using System.Security.Cryptography.X509Certificates;
 
-public record TrustSettings(
-    string? UserAnchors = null,
-    string? TrustAnchors = null,
-    string? TrustConfig = null,
-    string? AllowedList = null);
+namespace ContentAuthenticity;
 
+public enum OcspFetch
+{
+    All,
+    Active
+}
+
+public enum ThumbprintQuality
+{
+    Low,
+    Medium,
+    High
+}
+
+// [trust]
+public record TrustSettings(string? UserAnchors = null, string? TrustAnchors = null, string? TrustConfig = null, string? AllowedList = null)
+{
+    private static TrustSettings? __cachedSettings = null;
+
+    public static TrustSettings Default
+    {
+        get
+        {
+            if (__cachedSettings == null)
+            {
+                __cachedSettings = new TrustSettings();
+            }
+            return __cachedSettings;
+        }
+    }
+}
+
+// [verify]
 public record VerifySettings(
     bool VerifyAfterReading = true,
     bool VerifyAfterSign = true,
-    bool VerifyTrust = false,
-    bool VerifyTimestampTrust = false,
+    bool VerifyTrust = true,
+    bool VerifyTimestampTrust = true,
     bool OcspFetch = false,
+    bool RemoteManifestFetch = true,
     bool CheckIngredientTrust = true,
     bool SkipIngredientConflictResolution = false,
-    bool RemoteManifestFetch = true,
-    bool StrictV1Validation = false);
+    bool StringV1Validation = false,
+    bool Detailed = false)
+{
+    private static VerifySettings? __cachedSettings = null;
+
+    public static VerifySettings Default
```

Comment:
Why is this needed? The SDK doesn't need to cache the settings and the caller can cache on their side if needed.